acapy_wallet_upgrade/__main__.py

```diff
@@ -73,6 +73,14 @@ def config():
             "(mwst-as-stores) strategy."
         ),
     )
+    parser.add_argument(
+        "--limit",
```

Comment:
I think `limit` out of the context of the methods is a bit of an unclear term for this. `batch-size` would probably be a bit closer to what we might expect to see on the command line.